### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by meswan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,16 @@
...
+ - gradle
+    - This template contains the core files needed for Open Liberty and Gradle to run.
+ - maven
+    - This template contains the core files needed for Open Liberty and Maven to run.
+  - simple-rest-app	
+    - This template contains both Gradle and Maven core files to run a simple greeting REST endpoint.
+    - Once started, the endpoint can be reached at http://localhost:9080/api/resource/greeting
+    
+## Liberty Starter
+The [Liberty Starter](http://start.openliberty.io) page can be used to generate maven or gradle projects.
```
Comment (yeekangc) [inline README.md:15]: Is this live?  If not, we should remove this section.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,16 @@
-# project-templates
-Open Liberty Maven and Gradle project templates
+![](https://github.com/OpenLiberty/open-liberty/blob/master/logos/logo_horizontal_light_navy.png)
+
+This repository contains starter templates for Open Liberty.
+	
+## Templates
+ - gradle
+    - This template contains the core files needed for Open Liberty and Gradle to run.
```
Comment (yeekangc) [inline README.md:7]:
```suggestion
    - This project template contains the core set of files to build applications with Open Liberty using Gradle as the build tool.
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,16 @@
...
+ - maven
+    - This template contains the core files needed for Open Liberty and Maven to run.
```
Comment (yeekangc) [inline README.md:9]:
```suggestion
    - This project template contains the core set of files to build applications with Open Liberty using Apache Maven as the build tool.
```

File: README.md (hunk 1 above)
Comment (meswan) [reply README.md]: Removed!

File: README.md [hunk 4]
```diff
@@ -1,2 +1,16 @@
...
+  - simple-rest-app	
+    - This template contains both Gradle and Maven core files to run a simple greeting REST endpoint.
+    - Once started, the endpoint can be reached at http://localhost:9080/api/resource/greeting
```
Comment (yeekangc) [inline README.md]:
```suggestion
    - The sample REST endpoint can be reached at http://localhost:9080/api/resource/greeting.
```

File: README.md [hunk 5]
```diff
@@ -1,2 +1,16 @@
...
+
+This repository contains starter templates for Open Liberty.
+	
+## Templates
+ - gradle
+    - This template contains the core files needed for Open Liberty and Gradle to run.
+ - maven
+    - This template contains the core files needed for Open Liberty and Maven to run.
+  - simple-rest-app	
+    - This template contains both Gradle and Maven core files to run a simple greeting REST endpoint.
```
Comment (yeekangc) [inline README.md:11]:
```suggestion
    - This project template contains the core set of files to build applications with Open Liberty using Gradle or Maven with a simple REST endpoint included.
```

**Review by yeekangc — CHANGES_REQUESTED** [review]

**Review by yeekangc — APPROVED** [review]

**Pull request merged**